Dwolla.Client/HttpServices/DwollaHttpService.cs

```diff
@@ -15,15 +15,16 @@ public IDwollaClient Client
 		{
 			get => _client ?? DwollaConfiguration.Client;
 			set => _client = value;
-		}		
+		}
 		
 		private static TokenResponse _cachedToken;
+		private static DateTime _expiresAt;
 
-		private Task<string> getAccessTokenAsync;	
+		private readonly Func<Task<string>> _getAccessTokenAsync;
 
 		private async Task<string> GetAccessTokenAsync()
 		{
-			if (_cachedToken != null)//_cachedToken.ExpiresIn > now)
+			if (_cachedToken != null && _expiresAt > DateTime.Now)
```

Comment:
Use `DateTime.UtcNow` and rename expires at to `_expiresAtUtc` so it's clear everything is in UTC